scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -151,3 +165,102 @@ cd ..
 # The archives generated by `m0reportbug` are already compressed (.tar.gz).
 # compress them again to keep consistency with SB of other component.
 tar --remove-files -czf "${COMPONENT}_${bundle_id}.tar.gz" $HOSTNAME
+
+TEMP=( $( ls -ln "${COMPONENT}_${bundle_id}.tar.gz" ) )
+bundle_size=${TEMP[4]}
+
+binaries_size_limit() {
+    find $PWD/m0reportbug-cores/binaries/ -type f -exec grep -IL . "{}" \; | while read binfile; do
+        tar -czf "${binfile}.tar.gz" --absolute-names $binfile
+        TEMP=( $( ls -ln "${binfile}.tar.gz" ) )
+        size=${TEMP[4]}
+        rm "${binfile}.tar.gz"
+        if [[ $DIFF -gt $size ]]; then
+            DIFF=$(($DIFF - $size))
+            rm $binfile
+        else
+            DIFF="${DIFF/#-}"
```

Comment:
[Suggestion]
We enter into else means, we have reached within limit of max support bundle size allowed, correct ?
if yes. Then we can break the loop here to avoid un-necessary traversal of all the files.

Reply:
incorporated